### ISSUE TITLE
2015.11.02 - Issue Queue vs PR queue Priority

### ISSUE THREAD

**Issue by DanVanAtta** [triplea-game/triplea#258]
@veqryn  I request/recommend looking at the "Process Improvement" issues first. Any that you agree with outright can be unblocked and we can engage in parallel efforts. After unblocking those projects the priority would then in my view shift back to the PR queue.  All of the non "process improvement" issues are pretty much all parked, tracking tickets.
